Lib/dis.py

```diff
@@ -32,20 +32,30 @@ def _try_compile(source, name):
     return c
 
 def dis(x=None, *, file=None, depth=None):
-    """Disassemble classes, methods, functions, generators, or code.
+    """Disassemble classes, methods, functions, and other compiled objects.
 
-    With no argument, disassemble the last traceback.
+       With no argument, disassemble the last traceback.
```

Comment:
Align the docstring at the left side of `"""`. Seems this is never specified explicitly (likely because nobody could think of other way of formatting), but all multiline docstrings are aligned at the left side of `"""`.